lib/connection/bind_uploader.js

```diff
@@ -113,6 +114,13 @@ function BindUploader(options, services, connectionConfig, requestId) {
     if (data == null || data.toString() === '') {
       return '""'; 
     }
+    if (!isString(data)) {
+      if (data instanceof Date) {
```

Comment:
I'm wondering how about other types, especially binary?

Reply:
I've added binary and boolean columns to test